Shared/Components/ImageView.swift

```diff
@@ -25,6 +25,7 @@ private let imagePipeline = {
 //       - instead of removing first source on failure, just safe index into sources
 // TODO: currently SVGs are only supported for logos, which are only used in a few places.
 //       make it so when displaying an SVG there is a unified `image` caller modifier
+// TODO: probably don't need both `placeholder` modifiers
```

Comment:
I don't think you merged correctly as these aren't on main/my branch.

Reply:
yep. noticed. currently trying to figure out what happened

Reply:
Fixed it. Very odd.